doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -5,180 +5,160 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
+Processes can send and receive Kafka messages using message events.
+A {KIE_SERVER} can emit Kafka events when a process, case, or task completes.
 
-{PRODUCT} now supports {OPENSHIFT} version 4.6. In order to support this version, necessary changes in the operator bundle format are made.
-{OPENSHIFT} version 4.6 provides an extended update support. For more information, see https://access.redhat.com/support/policy/updates/openshift-eus?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG[Red Hat OpenShift Extended Update Support (EUS) Overview].
+//For more information about integration with Red Hat AMQ Streams or Apache Kafka, see {URL_INTEGRATING}#assembly-integrating-amq-streams[_{INTEGRATING_AMQ_STREAMS}_].
 
-IMPORTANT: {PRODUCT} {PRODUCT_VERSION} is supported only on {OPENSHIFT} version 3.11 and {OPENSHIFT} version 4.6 EUS. Due to the internal changes of the operator and API version, {OPENSHIFT} version 4.5 is not supported and this makes an exception to the general support policy for the Red Hat Middleware products.
-For more information, see https://access.redhat.com/articles/5115291[Support of Red Hat Middleware products and components on Red Hat OpenShift].
+=== Support for client roles in {RH-SSO}
 
-=== Support for LDAP failover
+When integrating {PRODUCT} with {RH-SSO}, you can enable support for client roles. By default, only realm roles are supported.
 
-When using LDAP authentication, {PRODUCT} on {OPENSHIFT} now supports failover to a different LDAP server if a server becomes unavailable.
+For more information about integration with {RH-SSO}, see {URL_INTEGRATING}#assembly-integrating-sso[_{INTEGRATING_SSO}_].
 
-For more information, see {URL_DEPLOYING_ON_OPENSHIFT}#operator-deploy-security-proc_openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_] and {URL_DEPLOYING_ON_OPENSHIFT}#template-deploy-ldap-openshift-templates-authoring-proc[_{DEPLOYING_OPENSHIFT_TEMPLATES}_].
+== {KIE_SERVER}
 
-=== Integration with new versions of products
+=== Running job failover
 
-{PRODUCT} on {OPENSHIFT} now supports integration with the following new product versions:
+If you have multiple {KIE_SERVER} nodes in cluster and one node fails while in a running state, the jobs in that instance are requested to another running node.
 
-* {AMQ} {AMQ_VERSION}
-* {AMQ_BROKER_LONG} {AMQ_BROKER_VERSION}
-* {RH-SSO} 7.4.1
-* {DATAGRID} {DATAGRID_VERSION}
+=== Singleton timer start node
 
-ifdef::PAM[]
+With the cluster extension, the jbpm engine now supports singleton timer start node in a process. When the extension is configured and one process is deployed in several instances, the timer start node will be only triggered once among all instances.
 
-=== Support for a customized Smart Router image
+== Spring Boot
 
-{PRODUCT} on {OPENSHIFT} now supports customized Smart Router images. You can extend Smart Router to provide routing specific to the needs of your environment.
+=== Plugable variable persistence
```

Comment:
I believe this should be called `Pluggable`.

Reply:
Done.